test/ios/app.test.ts

```diff
@@ -376,8 +417,11 @@ const PROJECT_FILES = {
           PRODUCT_VERSION: "1.0",
           USER_HEADER_SEARCH_PATHS: ["/~/node_modules/.generated"],
         },
+        communityAutolinkingScriptPath: undefined,
```

Comment:
Are these meant to be undefined?

Reply:
Yes, they are only required for older versions. See inline comment: https://github.com/microsoft/react-native-test-app/blob/627318d3b95b71fadc836779a41d8522b62beb71/ios/app.mjs#L75-L81